src/app/Instance/BridgeOverview/Components/BOSinkList.tsx

```diff
@@ -45,6 +45,8 @@ export const BOSinkList = (props: BOSinkListProps): JSX.Element => {
         ) : (
           <BODashboardTableView
             name={t("sinkConnector.sinkConnectors")}
+            deleteProcessor={(): void => {}}
+            onEditProcessor={(): void => {}}
```

Comment:
As per by previous comment, it looks wrong to have to set a `deleteProcessor` prop inside `BOSinkList`. Same happens for `BOSourceList`.